sha3_api.c

```diff
@@ -15,6 +16,12 @@ void sha3_init(sha3_var_t *c, size_t md_sz)
   size_t i;
 
   for (i = 0; i < 25; i++)
+  __loop__(
+    assigns(i, object_whole(c->st.d))
+    invariant(i <= 25)
+    invariant(forall(j, 0, i, c->st.d[j] == 0))
```

Comment:
This invariant is only needed to prove the post-condition. If you remove the post-condition, you can also remove the invariant.